schemas/schema.json

```diff
@@ -64,14 +64,8 @@
     }
   },
   "type": "object",
```

Comment:
Previously, `IOS|ANDROID|SHARED` were the only allowed properties in the root of the Localicipe. Now, we'll accept any collection name here. We'll still validate if the requested collections are present in the Localicipe, the only difference is that the `properties` property is now generated dynamically during runtime.